test/riak_kv_sweeper_SUITE.erl

```diff
@@ -234,10 +382,21 @@ test_scheduler_remove_participant(Config) ->
                                        end),
     ok = receive_msg({ok, failed_sweep, sweeper_callback_wait, WaitIndex}).
 
+%%
+%% Test that riak_kv_sweeper:sweep(...) queues requests if the
+%% sweep_concurrency limit has been reached.
+%%
+%% The test uses sweep participant sweeper_callback_wait that calls
+%% back a Fun before visiting an keys.
+%%
+%% The test cases uses the sweeper_callback_wait:before_visit to make
+%% a sweep request on an idle index when we have reached the max
+%% concurrency limit, and asserts that the requested sweep is queued.
+%%
 test_scheduler_queue(Config) ->
     Indices = ?config(vnode_indices, Config),
     new_sweep_participant(sweeper_callback_wait),
-    sweeper_meck_helper:backend_modules(async, _NumKeys=5000, _ObjecSizeBytes=0),
+    sweeper_meck_helper:backend_modules(async, _NumKeys=1000, _ObjecSizeBytes=0),
```

Comment:
I just noticed we misspell "Object" as "Objec" in a bunch of places all over this test suite.